snuba/datasets/entities/sessions.py

```diff
@@ -79,6 +91,127 @@ def plus_columns(
     ]
 )
 
+quantity = Column(None, None, "quantity")
+seq = Column(None, None, "seq")
+status = Column(None, None, "status")
+session_id = Column(None, None, "session_id")
+distinct_id = Column(None, None, "distinct_id")
+duration = Column(None, None, "duration")
+lit_nil = Literal(None, NIL_UUID)
+
+
+def eq(col: Column, lit: Literal) -> FunctionCall:
+    return FunctionCall(None, "equals", (col, lit))
+
+
+def neq(col: Column, lit: Literal) -> FunctionCall:
+    return FunctionCall(None, "notEquals", (col, lit))
+
+
+def _and(ex1: Expression, ex2: Expression) -> FunctionCall:
+    return FunctionCall(None, "and", (ex1, ex2))
+
+
+def _in(col: Column, args: Iterator[Literal]) -> FunctionCall:
+    return FunctionCall(None, "in", (status, FunctionCall(None, "tuple", tuple(args))))
+
+
+has_errors = FunctionCall(
+    None, "greater", (Column(None, None, "errors"), Literal(None, 0))
```

Comment:
Please use this.
https://github.com/getsentry/snuba/blob/master/snuba/query/conditions.py#L22